File: Программирование/18.04.2020/3-я пара/Задача 2/2.cpp
/**
Вычислить значения функций и вывести их на экран (результаты расчетов должны совпадать). 
z =2*sin^2*(3pi-2*a)*cos^2(5*pi+2*a)
y =1\4 - 1\4 * sin((5\2) * pi - 8a)
Программа сначала вводит значение а, выводит z и на следующей строке y.
(Нужно подключить заголовочный файл <cmath>).

Выполнил: Герасимов Дмитрий Александрович П1-18
*/

#include <iostream>
#include <cmath>
using namespace std;

int main() {
    double a;
    cin >> a;
    cout << 2 * pow(sin(3*M_PI - 2*a),2)*pow(cos(5*M_PI + 2*a),2) << endl
        << 1./4 - 1./4 * sin(5. / 2 * M_PI - 8 * a) << endl;
    
    return 0;
}

/**
Sample Input:

6.7
Sample Output:

0.495362
0.495362

*/

File: Алгоритмы/Лаба 1/2.py
'''
Брусник Вадим П1-18
Массив
Вводится число N, сформировать массив чисел от 0 до N.
Вывести этот массив.
Input:
    10
Output:
    0 1 2 3 4 5 6 7 8 9
'''
LEN = 100
mas = [LEN]
n = int(input())
mas = [int(i) for i in range(n)]
print(*mas)
